### ISSUE TITLE
Transparent box around the GUI

### ISSUE THREAD

**Issue by cirala** [Ulauncher/Ulauncher#262]
![render](https://user-images.githubusercontent.com/41735402/43257770-f24450d2-90d0-11e8-8603-fe0ba35e00b5.png)

**Steps to reproduce:**
Start Ulauncher
CTRL + SPACE to open up App Search
Note: Issue is present in Preferences as well

**Ulauncher version**:
4.1.0.r1

**Linux distro**:
Manjaro XFCE 17.1.11
**Desktop environment**:
XFCE

**Logs**:
https://gist.github.com/cirala/80319d1d533bbdf79a9d454d39e513e0

**Comment by cirala** [comment]
Issue solved, my Compositor (compton) seems to be making shadows around the window which causes this issue a simple exclude fixed it.

**Comment by lalitmee** [comment]
@cirala, what exclude? I am also facing the same issue.

**Comment by subdavis** [comment]
`compton --shadow-exclude 'class_g = Ulauncher'` did it for me.

**Comment by twnaing** [comment]
I have to use `blur-background-exclude` in picom

```conf
blur-background-exclude = [
  # others
  "class_g  = 'Ulauncher'"
]
```

**Comment by Nakamerica** [comment]
I have the same issue in gnome 42, Fedora 36... not sure how to fix this.

**Comment by lucamir** [comment]
I have the same problem on ubuntu 22.10.
Someone know how to fix it?

**Comment by Nakamerica** [comment]
Hi, I managed to fix this issue after a fair bit of trial and error. You just need to put the `-no-window-shadow` flag in the Exec line in the ulauncher.desktop file. 

Hope this helps, this fixed the issue for me.

**Comment by friday** [comment]
This is all covered by the [Troubleshooting](https://github.com/Ulauncher/Ulauncher/discussions/991) page now, which is linked to from the GitHub index page, the GitHub "create issue" listing page, the issue template as well as pinned in discussions and as of just recently the app preferences help page.

I also linked to the individual posts here for compton and picom, but I'm not sure it's needed with the flag.

In v6 I tried to make a change so this flag/setting wouldn't be needed. Haven't tried it out myself, but if anyone here would like to try I'd appreciate it (#1149).
